_posts/2015-06-26-ux-part-1.md

```diff
@@ -0,0 +1,69 @@
+
+*Attendees at the OME Users Meeting Paris 2014.*
+
+We share common basic development aims with most teams producing software: deploy functional software that allows users to achieve their tasks, and has ease of use, learnability and robustness. Simple aims, but as all developers know, somewhat tricky to achieve.
+
+The OME system is a classical example of an “academic expert system”, i.e. one designed by experts in a domain for use by other experts in that domain, and coming out of an academic environment. This is a very well trodden path and is well recognised in the UX/HCI community as having very particular issues and challenges associated with it. One of the biggest UX challenges with academic expert systems is getting from the old developer chestnut “it works for me ...” to the point where it also works for a broad range of users in a wide range of domains.
+
+Developers may be doing this for altruistic reasons, spreading the joy of their software, or more pragmatic reasons, such as attracting grant funding. Either way, going from a narrow, constrained, tidy focus to a relatively open-ended, unconstrained environment can be quite daunting.
+
+##Moving to a Broader User Base
+
+The starting point is looking at who all the current and potential users are, working out how to get them on board, and then how to satisfy their requirements. This often involves considerable change in the user interface to enable non- or not-so-expert users to understand and use the software. The OME project established much of the groundwork drawing on the expertise of ethnographic-focused HCI specialists, and then forming the UX team within the project to carry through the work.
+
+The primary users of OME are bench Life Scientists, but our user base has rapidly expanded to include digital pathology, teaching, publishing and a number of other domains. This expansion is continual and open-ended. But OME does not just have the people who use the clients, scientists and so on, as users, we also have facility managers and system administrators in institutions using OMERO. 
+
+Then in addition to this, as an open source project, OME also effectively has a lot of other developers as users of the system. And to add another layer, being open source, this is very much a two-way street, so the developer-users are also part of the team as well as being users. A slightly tangled web but very fruitful if handled well!
+
+The focus of this series of posts is on the scientist user and the GUI clients, OMERO.insight and OMERO.web, where the primary UX team involvement is. 
+
+It may sound obvious, but the client user interface is the only way the majority of users have to interact with any computer system. No matter how wonderful the system is behind the scenes, the typical user only gets to use what they can see and interact with via the user interface. So in effect, the UI ultimately determines the functionality of the software for most users.
```

Comment:
Very minor nitpick, but I'd be inclined to use client GUI throughout once you've introduced it rather than sometimes saying client user interface as here.